src/GitExtensions.Extensibility/tools/Download-GitExtensions.ps1

```diff
@@ -0,0 +1,229 @@
+param(
+  [Parameter(Mandatory=$true)]
+  [string] $ExtractRootPath, 
+  [Parameter(Mandatory=$true)]
+  [string] $Version, 
+  [ValidateSet('GitHub','AppVeyor', ignorecase=$False)]
+  [string] $Source = "GitHub"
+)
+
+$LatestVersionName = "latest";
+
+function Test-LocalCopy 
+{
+	Param(

```

Comment:
We may wish to use the same .editorconfig and use space instead of tabs

Reply:
👍